StockAnalyzerService/StockAnalyzerService/Service/StockAnalyzer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using StockAnalyzerService.Model;
+using StockAnalyzerService.Service;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace StockAnalyzerService.Service {
+	public class StockAnalyzer : IStockAnalyzer {
+		public IHttpClientFactory clientFactory;
+		public IHttpCalls httpCalls;
+		public StockAnalyzer(IHttpClientFactory _clientFactory, IHttpCalls _httpCalls) {
+			clientFactory = _clientFactory;
+			httpCalls = _httpCalls;
+		}
+
+		public async Task<List<StockMetadata>> GetStocksWithUsers() {
+			try {
+				string urlParameters = "stock/user";
```

Comment:
It is stocks/users now. Do you need a "/" in the front like "/stocks/users"?

Reply:
That extra slash is in the configure services method in Program.cs